vac/raw/eth-secpm.md

```diff
@@ -824,6 +824,33 @@ These credentials MUST use the digital signature key pair associated to the Ethe
 - A Delivery Service storage system MUST verify KeyPackages before storing them.
 - Interaction diagrams involving the DS do not change.
 
+## Consideration related to the onchain component of the protocol
+- Each group is managed by a separate smart contract.
+
+- Below follows a description of Alice adding Bob as a member of a new group.
```

Comment:
I'm not sure that is correct order. I've added comments further on each item.

Some context: 
- At first we have two pair of long-term key - the one is associated to ethereum wallet and the second is from openmls (it is used for sign the key package on openmls side)
- The current implementation of the key package includes the signature and the public key, so verification takes place without additional data, only the key packet is sufficient. https://github.com/openmls/openmls/blob/96c38a806f9c706d2cf67566c9c846eee3ac4430/openmls/src/key_packages/mod.rs#L179-L182